app/script/cryptography/CryptographyRepository.coffee

```diff
@@ -212,28 +212,28 @@ class z.cryptography.CryptographyRepository
       return undefined
 
   _encrypt_generic_message_for_new_sessions: (user_client_map_for_missing_sessions, generic_message) =>
-    return new Promise (resolve) =>
-      if Object.keys(user_client_map_for_missing_sessions).length
-        @get_users_pre_keys user_client_map_for_missing_sessions
-        .then (user_pre_key_map) =>
-          @logger.info "Fetched pre-keys for '#{Object.keys(user_pre_key_map).length}' users.", user_pre_key_map
+    if Object.keys(user_client_map_for_missing_sessions).length is 0
+      return Promise.resolve []
+
+    @get_users_pre_keys user_client_map_for_missing_sessions
+    .then (user_pre_key_map) =>
+      future_sessions = []
 
-          future_sessions = []
+      @logger.info "Fetched pre-keys for '#{Object.keys(user_pre_key_map).length}' users.", user_pre_key_map
```

Comment:
Do you really want to make this an `info` message? Info loggings are quite verbose.